### ISSUE TITLE
Windows 7 not supported? Error: 0xc0000005

### ISSUE THREAD

**Issue by ElectroMyStyle** [amnezia-vpn/amnezia-client#490]
**Describe the bug**
During the installation of the application on Windows 7, an error occurs - `0xc0000005`. When trying to uninstall an application through the installer file, the same error occurs. After installation, the application files are installed in the folder `C:/Program Files/AmneziaVPN`, but when trying to launch the application `AmneziaVPN.exe `an error occurs - `0xc0000005`

**To Reproduce**
1. Downloaded the installer version 4.1.2 from the [Release section](https://github.com/amnezia-vpn/amnezia-client/releases)
2. Launched the installation file on Windows 7 Ultimate SP1
3. Apparently, upon completion of the installation, the application file does not run, which gives an error - `0xc0000005`
4. See the error in the screenshot

**Expected behavior**
I expect the application to work

**Log files**
The log file is attached. [InstallationLog.txt](https://github.com/amnezia-vpn/amnezia-client/files/13849967/InstallationLog.txt)

**Screenshots**
![Amnezia Install](https://github.com/amnezia-vpn/amnezia-client/assets/5159210/2b360dff-7f9d-487b-a8f4-e59dab80a7b2)
![Amnezia Uninstall](https://github.com/amnezia-vpn/amnezia-client/assets/5159210/0c718723-92b1-4c04-9612-0d2d1eed59ec)

**Desktop (please complete the following information):**
 - OS: Windows 7 Ultimate SP 1

**Additional context**
I examined the error log in Windows and found two errors:
1. Event ID: 7000. The AmneziaVPN-service service failed to start due to the following error: 
The service did not respond to the start or control request in a timely fashion.
2. Event ID: 7009. A timeout was reached (30000 milliseconds) while waiting for the AmneziaVPN-service service to connect.

**Comment by Bushroott** [comment]
Windows 7 is not supported because the version of Qt used by the developers does not support it.
Why they use the latest versions of libraries remains a mystery. There is no technical need for this. New doesn't mean better.

**Comment by Healertime** [comment]
Is there an earlier release that will work?

**Comment by Bushroott** [comment]
> Is there an earlier release that will work?

2.1.2

**Comment by pokamest** [comment]
We are working on Windows 7 support, I hope it will be finished in couple weeks

**Comment by Bushroott** [comment]
Nothing has changed in the new version 4.5.3
The program does not work in Windows 7
As before, there is no OS version check before installation.

**Comment by maximlunegov** [comment]
> As before, there is no OS version check before installation.

And I think there should be a note in the release about the compatible OS version.

**Comment by deksden** [comment]
@pokamest : any news for windows 7 support? Latest released (4.6+) did not install/work on win7, old release 2.1.2 did not work with modern configs. 


here is patched QT6 working on Win7: https://github.com/crystalidea/qt6windows7

**Comment by mir-math** [comment]
I have unzipped 4.7.0 installation file into a folder, replaced all QT6 dll-s to those from (https://github.com/crystalidea/qt6windows7)  and client still doesn't start. Says:
Entry point  ?vulkanInstance@QWindow@@QEBAPEAVQVulkanInstance@@XZ not found in DLL Qt6Gui.dll. 

Looks like we are facing known issue from that project "QRhi using DirectX 11/12 is not ported".  QTDesigner in the same folder works, so qt6windows7's DLL-s are working, but this is not enough  for  AmneziaVPN.

**Comment by deksden** [comment]
I hoping that this issue is important and will be resolved!

**Comment by Maggistrator** [comment]
Agreed. Some relatives of mine are unwilling to migrate from Windows 7, so I need Windows 7 support badly to let them connect to my Amnezia server

**Comment by freethinked** [comment]
Version 4.8.3.1, Windows 7 still doesn't work

**Comment by Nethius** [comment]
currently the application does not support windows 7
